functions/divisional.inc

```diff
@@ -302,21 +294,14 @@ function ConComListBuildSearch($page) {
 function ConComListBuild() {
   foreach( $GLOBALS['Departments'] as $kdep => $dep) {
     if( $dep['Division'] != $kdep ) {
-      if( !empty($dep['OtherEmails'])) {
-        $addEmail=explode("|", $dep['OtherEmails']);
-        foreach($addEmail as $tmp) {
-          $dep['Email'] .= "<br />" . $tmp;
-        }
-        unset($tmp);
-      }
```

Comment:
will this leave this stanza open?

Reply:
meaning an unmatched {..}?  no i think it is ok.  Otherwise I don't quite understand...
It looks like it was taking all the Email addresses, exploding them into an array and then appending them with <br />.  which i now do with the implode below.